src/tools/rustfmt/src/patterns.rs

```diff
@@ -193,6 +193,13 @@ impl Rewrite for Pat {
                     None
                 }
             }
+            PatKind::Never => {
+                if 1 <= shape.width {
+                    Some("!".to_owned())
+                } else {
+                    None
+                }
+            }
```

Comment:
Can you return `None` here to make this always a no-op? I think the policy is to avoid implementing style for new features, and leave the formatting for later.

Reply:
Even in such an obvious case? I copy-pasted the `_` case here. [This](https://github.com/rust-lang/style-team/blob/main/nightly-style-procedure.md) seems to imply "you can change rustfmt for a nightly feature but don't break anything".

Reply:
> Initial PR(s) implementing new syntax filed against rust-lang/rust should generally leave the rustfmt subtree untouched. In cases that that need to modify rustfmt (for example, to fix compiler errors that come from adding new variants to AST representation), they should modify rustfmt in such a way to keep existing formatting verbatim.

Reply:
Ohh, I misread that. I understood it as "in such a way that anything that doesn't use the new syntax stays formatted the same"